plugins/catalog-backend-module-opa-entity-checker-processor/README.md

```diff
@@ -0,0 +1,26 @@
+# Backstage OPA Backend Module Entity Processor
+
+When it comes to validating entries the Backend and Frontend plugins are great combo to bring validation errors to the user. However, this doesn't allow you to have the big picture of hom many entities in your catalog have fail the validation.
+
+Then you can search entities by annotation using the API and get a list of all the entities that fails the validation.
+
+```http request
+GET http://localhost:7007/api/catalog/entities/by-query?filter=metadata.annotations.open-policy-agent/entity-checker-violations-status=error
+Content-Type: 'application/json'
+Authorization: Bearer {{BACKSTAGE_TOKEN}}
+```
+
+## Why annotation and Not Status updates
+
+Editing errors leads to new entities not to be process, which is not always what we want, some "warning" are acceptable and we do not want to break processing and add additional errors message since we already provide frontend plugin to display those error message in a nicer and mor compact way.
```

Comment:
```suggestion
When errors are edited, it can result in new entities not being processed. This is not always desirable as some warnings are acceptable and we do not want to disrupt processing or add additional error messages. We already provide a frontend plugin to display these error messages in a more user-friendly and compact way.
```